packages/shortest/README.md

```diff
@@ -73,6 +79,42 @@ shortest('Login to the app using username and password', {
 });
 ```
 
+## Test Dependencies
+You can create reusable test steps and chain them together. This helps organize complex test flows and maintain state between related tests:
+
+```typescript
+import { shortest, after } from '@antiwork/shortest';
+
+// Define reusable test steps
+const login = shortest('User can login', { 
+  email: 'test@example.com',
+  password: 'password'
+});
+
+const settings = shortest('Click settings button');
+
+// Chain tests with dependencies
+after(login, settings)
+  .shortest('Change language to Arabic')
```

Comment:
```suggestion
  .shortest('Language can be changed to Arabic')
```

Reply:
instead of expect the developer can use this approach